rules/splice.bzl

```diff
@@ -36,6 +36,14 @@ def _bitstream_splice_impl(ctx):
     tmpsrc = ctx.actions.declare_file("{}.tmpsrc.bit".format(ctx.label.name))
     ctx.actions.symlink(output = tmpsrc, target_file = ctx.file.src)
 
+    splice_env = {k: v for (k, v) in ENV.items()}
+
+    # Enable support for RAMB18 in `updatemem`. This environment variable seems
+    # completely undocumented; I only found a reference to it on a Xilinx forum:
+    # <https://support.xilinx.com/s/question/0D52E00006iHlJXSA0>. I also needed
+    # to bump MMI minor version 1 before it would accept RAMB36 and RAMB18.
+    splice_env["UPDATEMEM_ALLOW_RAMB18"] = "1"
```

Comment:
I believe this actually isn't necessary anymore, and the environment variable doesn't do anything for Vivado 2020.1. Instead, the key is that higher MMI minor version.

Reply:
Good catch! I think setting this environment variable changed the error I was getting, so I thought it was necessary. Just verified that it's unnecessary.